vite.config.ts

```diff
@@ -117,5 +121,10 @@ export default meta;
 `
   );
 
+  fs.writeFileSync(
+    './src/data/GithubStats.json',
+    JSON.stringify(codeLanguages)
+  );
+
```

Comment:
The file will be downloaded every time a developer runs the code. This behavior is undesirable.
The code should not actively download the file; let an external system handle that.